apps/cms.c

```diff
@@ -802,6 +802,9 @@ int cms_main(int argc, char **argv)
     if ((operation & SMIME_IP) == 0 && contfile != NULL)
         BIO_printf(bio_err,
                    "Warning: -contfile option is ignored for the given operation\n");
+    if (operation != SMIME_ENCRYPT && *argv != NULL)
+        BIO_printf(bio_err,
+                   "Warning: recipient certificate file parameters ignored for operation other than -encrypt\n");
```

Comment:
Hmm, it looks to me as if this message used to be issued when `operation == SMIME_ENCRYPT` and `*argv == NULL` - what am I missing here?